src/db/migrations/test/migration2.tests.js

```diff
@@ -0,0 +1,146 @@
+    resultSetterAddress: 'qWG4tBKgscVxkRGwRZdBt2Whkb66VD5g9h',
+    startTime: 1529540100,
+    endTime: 1530389700,
+    resultSetStartTime: 1530389700,
+    resultSetEndTime: 1530403200,
+    consensusThreshold: '10000000000',
+    name: 'Winner of the Bay Area Best Ramen Bar Grand Prix',
+    options: ['Invalid', 'Miso-ya', 'Ramen Do-jo', 'Ippudo SF', 'Shalala Ramen'],
+    optionIdxs: [0, 1, 2, 3, 4],
+    amounts: ['0', '812000000', '0', '19990000000', '100000'],
+    resultIdx: 3,
+    status: 'WITHDRAW',
+    token: 'QTUM',
+    hashId: null,
+    language: 'zh-Hans-CN',
+  };
+
+  const dOracle = {
+    blockNum: 223343,
+    txid: 'f306b48d2197402dd801bf8a0c768f52bb5cd27b12da773bbadca680864aad64',
+    version: 0,
+    address: '596b41260670dc5d0187d0b6f92773794509c8bc',
+    topicAddress: '64a2cc9a78f2faff6a7401e393f3a90731704ab6',
+    amounts: fill(Array(5), '0'),
+    optionIdxs: [0, 1, 2, 4],
+    endTime: 1538387296,
+    consensusThreshold: '11000000000',
+    name: 'Winner of the Bay Area Best Ramen Bar Grand Prix',
+    options: ['Invalid', 'Miso-ya', 'Ramen Do-jo', 'Ippudo SF', 'Shalala Ramen'],
+    resultIdx: null,
+    startTime: 1538383696,
+    status: 'WITHDRAW',
+    token: 'BOT',
+    language: 'zh-Hans-CN',
+  };
+  await DBHelper.insertOracle(db.Oracles, cOracle);
+  await DBHelper.insertOracle(db.Oracles, dOracle);
+}
+
+async function insertVotes(db) {
+  const resultset = {
+    blockNum: 223343,
+    txid: 'f306b48d2197402dd801bf8a0c768f52bb5cd27b12da773bbadca680864aad64',
+    version: 0,
+    oracleAddress: '21255aa2ece777d8d7b9572fe018761aa3a6dde1',
+    voterAddress: 'qJHp6dUSmDShpEEMmwxqHPo7sFSdydSkPM',
+    optionIdx: 3,
+    amount: '10000000000',
+    token: 'BOT',
+    topicAddress: '64a2cc9a78f2faff6a7401e393f3a90731704ab6',
+  };
+  const vote = {
+    blockNum: 223345,
+    txid: '35d7745be0da899675ec376cc50f051a25642afc38fd3f718541f3e05638deca',
+    version: 0,
+    oracleAddress: '596b41260670dc5d0187d0b6f92773794509c8bc',
+    voterAddress: 'qJHp6dUSmDShpEEMmwxqHPo7sFSdydSkPM',
+    optionIdx: 3,
+    amount: '10000000000',
+    token: 'BOT',
+    topicAddress: '64a2cc9a78f2faff6a7401e393f3a90731704ab6',
+  };
+  const bet = {
+    blockNum: 166342,
+    txid: 'e7cef57e9202819ebbc7db919f42aaec038cbbd378c608bacb5f02d318d4021f',
+    version: 0,
+    oracleAddress: '596b41260670dc5d0187d0b6f92773794509c8bc',
+    voterAddress: 'qMmN4PNBmHvCcUAfQBHkGZsYrGkovzo8W2',
+    optionIdx: 4,
+    amount: '100000',
+    token: 'QTUM',
+    topicAddress: '64a2cc9a78f2faff6a7401e393f3a90731704ab6',
+  };
+  await db.Votes.insert(resultset);
+  await db.Votes.insert(vote);
+  await db.Votes.insert(bet);
+}
+
+async function initDB(db) {
+  if (fs.existsSync(path.join(__dirname, './mock/votes.test.db'))) fs.unlinkSync(path.join(__dirname, './mock/votes.test.db'));
+  if (fs.existsSync(path.join(__dirname, './mock/oracles.test.db'))) fs.unlinkSync(path.join(__dirname, './mock/oracles.test.db'));
```

Comment:
lets delete the test DBs after the test is done so we dont have the files there.